patient/models.py

```diff
@@ -0,0 +1,78 @@
+import hashlib
+
+from django.core.validators import MaxValueValidator, MinValueValidator
+from django.db import models
+
+from common.base_models import ImmutableBaseModel, CurrentBaseModel
+
+
+class Patient(CurrentBaseModel):
+    """
+    Patient record, ties all patient related info together.
+    """
+
+    identifier = models.CharField(max_length=50, null=False)
+    id_type = models.CharField(max_length=50, null=False)
+
+    class Meta:
+        ordering = ['-created']
+
+    @property
+    def patient_anon_id(self):
```

Comment:
Do we really still need this? I would just drop it since we originally created it in the sense of patients pre-registering online on a public facing website. Also see comment below.

Reply:
Yeah, probably not. I'll remove it in the next iteration.